notes/returned-mail-received.vbs

```diff
@@ -55,26 +56,36 @@ changelog_display
 'END CHANGELOG BLOCK =======================================================================================================
 
 'THE SCRIPT-----------------------------------------------------------------------------------------------------------------
-'Connects to BLUEZONE
-EMConnect ""
-'Grabs the MAXIS case number
-CALL MAXIS_case_number_finder(MAXIS_case_number)
+
+EMConnect ""                                        'Connecting to BlueZone
+CALL MAXIS_case_number_finder(MAXIS_case_number)    'Grabbing the CASE Number
+call Check_for_MAXIS(false)                         'Ensuring we are not passworded out
+back_to_self                                        'added to ensure we have the time to update and send the case in the background
 
 '-------------------------------------------------------------------------------------------------DIALOG
-Dialog1 = "" 'Blanking out previous dialog detail
-'Running the initial dialog
-BeginDialog Dialog1, 0, 0, 211, 85, "Returned Mail"
-  EditBox 55, 5, 40, 15, MAXIS_case_number
-  EditBox 155, 5, 50, 15, date_received
-  DropListBox 90, 25, 115, 15, "Select One:"+chr(9)+"forwarding address in MN"+chr(9)+"forwarding address outside MN"+chr(9)+"no forwarding address provided"+chr(9)+"no response received", ADDR_actions
-  EditBox 90, 45, 115, 15, worker_signature
+Dialog1 = "" 'Running the initial dialog
+BeginDialog Dialog1, 0, 0, 221, 215, "RETURNED MAIL PROCESSING"
+  EditBox 60, 5, 50, 15, MAXIS_case_number
+  EditBox 60, 25, 50, 15, date_received
+  DropListBox 105, 65, 110, 15, "Select:"+chr(9)+"forwarding address in MN"+chr(9)+"forwarding address outside MN"+chr(9)+"no forwarding address provided"+chr(9)+"no response received", ADDR_actions
+  DropListBox 100, 150, 110, 15, "Select:"+chr(9)+"YES"+chr(9)+"NO-attempted"+chr(9)+"NO-not attempted", client_contact_dropdown 'TO DO question do we want more HERE'
```

Comment:
Looks like this only adds this to the CASE NOTE with no functionality support or other details. I am not sure I understand the purpose here.

Reply:
I can create an inhibiting error that we use client contact if they select yes